### DISCUSSION TITLE
srsRAN_Project 5G SA COTS UE NOT Registering -Quectel RM500q

### DISCUSSION THREAD

**Discussion by SAIRISAN123** [srsran/srsRAN_Project#482] [General Help] [answered]
![Screenshot from 2024-02-19 16-45-57](https://github.com/open5gs/open5gs/assets/118654165/bf5ca5db-aa44-4861-858d-3ec58f5e474c)

I have done all configuration and I am using B210 and I added Slicing part too. But  RM500q not registering to network. I have added the APN to the RM500q with AT command. But why this not connecting? Anyone help. 

I already tested 4G and 5G UERANSIM successfully.

**Comment by pgawlowicz** [comment]
could you provide ngap pcap?

**Reply by SAIRISAN123** [reply]
[NGAP Wireshark.zip](https://github.com/srsran/srsRAN_Project/files/14343810/NGAP.Wireshark.zip)

Thank you for the reply here is my pcap file.

**Reply by pgawlowicz** [reply]
could you share your gnb config as well?

**Reply by SAIRISAN123** [reply]
[gNB and Slicing config.tar.gz](https://github.com/srsran/srsRAN_Project/files/14345640/gNB.and.Slicing.config.tar.gz)

I have added my gNodeB , Slicing , and NF config sample files.  I am using sst 2 and sd 2

**Reply by SAIRISAN123** [reply]
![Screenshot 2024-02-21 at 10-17-14 Open5GS 2 7 0](https://github.com/srsran/srsRAN_Project/assets/118654165/14f763d4-0dba-46eb-aca8-3a5aac5c56d5)


![Screenshot 2024-02-21 at 10-17-41 Open5GS 2 7 0](https://github.com/srsran/srsRAN_Project/assets/118654165/c3fc285f-428a-4aea-92db-fcc1d85aab3d)

@pgawlowicz  here is my subscriber Information

**Reply by pgawlowicz** [reply, marked as answer]
it should be correct. Could you provide also gnb logs?

**Reply by SAIRISAN123** [reply]
[Logs.zip](https://github.com/srsran/srsRAN_Project/files/14358917/Logs.zip)

I have added gNB log and Other Core network sample log

No any response regarding UE connection in gNB log.

**Reply by pgawlowicz** [reply]
could you provide gnb logs, the file is at /tmp/gnb.log

**Reply by SAIRISAN123** [reply]
[GNB logs.zip](https://github.com/srsran/srsRAN_Project/files/14377044/GNB.logs.zip)


I have added my gnb logs

**Comment by pgawlowicz** [comment]
the gnb log does not contain any UE attach attempt. Did you try to connect the UE?

**Reply by SAIRISAN123** [reply]
[gnB log.tar.gz](https://github.com/srsran/srsRAN_Project/files/14400524/gnB.log.tar.gz)


Sorry for that. Now I have added the two gnB log files captures.

**Reply by pgawlowicz** [reply]
Still, there is no UE attach attempt in this log. but also there is a lot of RF issues: ``Real-time failure in RF``, you first need to solve those. You might try to run this script: https://github.com/srsran/srsRAN_Project/blob/main/scripts/srsran_performance
to put the host CPU in the performance mode.

**Reply by SAIRISAN123** [reply]
Hi @pgawlowicz I have connected now samsung s8 plus tablet. Like you said, not issue with config. I have re installed open5gs. It worked fine.

**Comment by fahad-ece** [comment]


**Reply by SAIRISAN123** [reply]
Hi @ff187 I am using linux. first you need to make your quectel to support your APN. Install minicom and use "**AT+CGDCONT=1,"IP","srsran**" command to do that. here srsran is your APN. you will receive OK as ACK. try this . if this didn't work. send the image of  amf status like first image I have posted here.

**Reply by chonghua1** [reply]
> Hi @ff187 I am using linux. first you need to make your quectel to support your APN. Install minicom and use "**AT+CGDCONT=1,"IP","srsran**" command to do that. here srsran is your APN. you will receive OK as ACK. try this . if this didn't work. send the image of amf status like first image I have posted here.

Hi @SAIRISAN123, could you please share how to set the slice config at Quectel module?
